requirements.in

```diff
@@ -0,0 +1,5 @@
+Django==1.11.15
+edx-opaque-keys==0.4.4
+pylint==1.9.3
+pycodestyle==2.4.0
+rocketchat_API==0.6.21
```

Comment:
Bumpversion requirement is missing. @andrey-canon

Reply:
ok